crates/holochain/src/sweettest/sweet_conductor_config.rs

```diff
@@ -3,44 +3,256 @@ use std::sync::Arc;
 use holochain_conductor_api::{conductor::ConductorConfig, AdminInterfaceConfig, InterfaceDriver};
 use kitsune_p2p::KitsuneP2pConfig;
 
+tokio::task_local! {
+    static RENDEZVOUS: DynSweetRendezvous;
+}
+
+/// How conductors should learn about each other / speak to each other.
+/// Just a bootstrap server in tx2 mode.
+/// Signal/TURN + bootstrap in tx5 mode.
+pub trait SweetRendezvous: 'static + Send + Sync {
```

Comment:
I think SweetRendezvous and friends deserve their own file, sil vous plait

Reply:
Will do :+1: